mosip-vid-policy.json

```diff
@@ -13,8 +13,8 @@
             "vidType": "Temporary",
             "vidPolicy": {
                 "validForInMinutes": 30,
-                "transactionsAllowed": 1,
-                "instancesAllowed": 20,
+                "transactionsAllowed": 2,
+                "instancesAllowed": 5,

```

Comment:
@kameshsr  - based on the previous conversation https://github.com/mosip/mosip-config/pull/7059#discussion_r1721327128

its changed to "2" becoz of this bug - https://mosip.atlassian.net/browse/MOSIP-35270

Are we going with this as a known issue? @vadherhemant 
cc: @vishwa-vyom

Reply:
https://mosip.atlassian.net/browse/MOSIP-35282
@ase-101 Above bug is not for vid policy issue. this is correct one.

Reply:
had a call with @kameshsr .  understood that earlier value was 1 by which API tests for Resident Service was failing.  Same is the failure when trying it out manually.  After making the policy change, this was tested in dev-staging env and APIs for RS are passing.  Not sure if this is the right solution, but having a fix is desired to have.

Reply:
since this a minor change, mark this comment as resolved